src/sentry/lang/javascript/processor.py

```diff
@@ -545,7 +545,10 @@ def preprocess_step(self, processing_task):
 
     def handles_frame(self, frame, stacktrace_info):
         platform = frame.get("platform") or self.data.get("platform")
-        return settings.SENTRY_SCRAPE_JAVASCRIPT_CONTEXT and platform in ("javascript", "node")
+        has_context_line = bool(frame.get("context_line"))
+        # if the frame already has a context_line, we consider it already symbolicated
+        # and therefore skip the processor handling it
+        return (not has_context_line) and settings.SENTRY_SCRAPE_JAVASCRIPT_CONTEXT and platform in ("javascript", "node") and is_valid_frame(frame)
```

Comment:
I'd have to check again to be 100% sure, but I think including `is_valid_frame` here means you can take it out of the preprocess step.